MODELS.md

```diff
@@ -0,0 +1,122 @@
+# Models
+
+A repository for the available models for AllenNLP.  While we highlight a particular model for
+each task on https://allennlp.org/models we often have other trained models that might
+work better for a particular application.
+
+
+## Machine Comprehension
+
+### [bidaf-model-2017.09.15-charpad.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/bidaf-model-2017.09.15-charpad.tar.gz) (44 MB)
+
+Based on [BiDAF (Seo et al, 2017)](https://www.semanticscholar.org/paper/Bidirectional-Attention-Flow-for-Machine-Comprehen-Seo-Kembhavi/007ab5528b3bd310a80d553cccad4b78dc496b02)
+
+```
+$ docker run allennlp/allennlp:v0.6.1 \
```

Comment:
Maybe remove these docker commands. They are already on the website, aren't available for all of the models, and will get stale really fast.

Reply:
They work indefinately because they're versioned.  It provides a record of the last version we tested the model on.  This will be particularly helpful if we include any models from outside AllenNLP.

Reply:
Yes I know that, I meant that it's just duplicating something from somewhere else, meaning it's another thing we have to update when we bump versions.

Reply:
Sure, but the same argument applies to having the models here and having performance numbers here.  I do plan to reduce the amount of duplication next quarter.

Reply:
It also gives me a quick "test case" that confirms the URL is accurate and I can actually make predictions with the model.

Reply:
Where else do we have a complete list of available models, sizes and download links for all models? But sure, leave it if you think it's better.